lib/ops/opsservice/instructions.go

```diff
@@ -76,7 +76,7 @@ echo "$(date) [INFO] Install agent will be using ${TMPDIR:-/tmp} for temporary f
 %v
 {{.service_user_env}}={{.service_uid}} \
 {{.service_group_env}}={{.service_gid}} \
-{{.gravity_bin_path}} {{if .devmode}}--insecure{{end}} --debug join {{.ops_url}} \
+{{.gravity_bin_path}} {{if .devmode}}--insecure{{end}} --debug --httpprofile=localhost:0 join {{.ops_url}} \
```

Comment:
I had planned to keep profiling on permanently but need to figure out if this really affects performance. Otherwise, it is a major help in troubleshooting.

Reply:
It's something to be careful of too, since customers tend to scan the software, and don't tend to like debug endpoints, even if they're bound to localhost.